js/date-filter.js

```diff
@@ -25,8 +25,13 @@ function DateFilter(elm) {
     oncomplete: this.validate.bind(this)
   });
 
+  if (this.$minDate.attr('data-min-date')) {
```

Comment:
Ah okay so my radio button was set to "this year" so I had mistakenly thought that was default. I've taken out default values for date input fields.